sdk/core/core-https/karma.conf.js

```diff
@@ -100,7 +100,7 @@ module.exports = function(config) {
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: true,
+    singleRun: false,
```

Comment:
was this intended to be changed? I'm assuming it was altered for testing.

What I usually do when I need to debug is

```
npx karma start --browsers=Chrome --single-run=false
```

Reply:
We have `--single-run` in the `unit-test:browser`/`integration-test:browser` npm scripts so it's less to type when we need debug with just `npx karma start --browsers=Chrome` but I can revert since it's unrelated change.

Reply:
I am cool with whatever is best here, I just thought it was a debug thing creeping into the main test path